BrowserKit/Sources/OnboardingKit/Models/OnboardingLinkInfoModel.swift

```diff
@@ -0,0 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+public struct OnboardingLinkInfoModel {
+    let title: String
+    let url: URL
```

Comment:
Should we should add a a11y id here as well?

Reply:
I though so too, but I was following the current onboarding implementation and it uses and interesting approach. It takes the root and appends different strings. I'm leaving here some examples

https://github.com/mozilla-mobile/firefox-ios/blob/f8735bb93873e7f7446e81baf31edc46afd4df4d/firefox-ios/Client/Frontend/Onboarding/Protocols/OnboardingCardViewController.swift#L69

https://github.com/mozilla-mobile/firefox-ios/blob/f8735bb93873e7f7446e81baf31edc46afd4df4d/firefox-ios/Client/Frontend/Onboarding/Protocols/OnboardingCardViewController.swift#L86

https://github.com/mozilla-mobile/firefox-ios/blob/f8735bb93873e7f7446e81baf31edc46afd4df4d/firefox-ios/Client/Frontend/Onboarding/Protocols/OnboardingCardViewController.swift#L77